lib/charms/mongodb/v1/mongodb_backups.py

```diff
@@ -141,13 +141,13 @@ def _on_s3_credential_changed(self, event: CredentialsChangedEvent):
             return
 
         if not self.charm.db_initialised:
-            self._defer_action_with_info_log(
+            self._defer_event_with_info_log(
```

Comment:
do we need to test here that event is not an action?

Reply:
IMO no